src/Runtime/CMakeLists.txt

```diff
@@ -58,8 +58,16 @@ set_target_properties(ExecutionSession
   )
 
 pybind11_add_module(PyRuntime PyExecutionSession.cpp)
+target_compile_definitions(PyRuntime
```

Comment:
Can you add some of the explanation of why these are added manually here? That way people looking at the cmake will know that they compile definitions are needed and why.

Reply:
I forgot to put the comments in the commit message here. I will add them in the next PR.